### PR TITLE
Validate ID for explicitly created Namespaces

### PR DESCRIPTION
**Pull request by dimas-b**

### CODE REVIEW

**Review by ajantha-bhat — DISMISSED** [review]

File: servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/tests/AbstractRestNamespace.java [hunk 1]
```diff
@@ -108,6 +108,7 @@ public void testNamespacesRetrieval() throws BaseNessieClientServerException {
     Namespace four = createNamespace.apply("one.two");
     for (Namespace namespace : Arrays.asList(one, two, three, four)) {
       soft.assertThat(namespace).isNotNull();
+      soft.assertThat(namespace.getId()).isNotNull();
```
Comment (snazy) [inline servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/tests/AbstractRestNamespace.java:111]: Maybe also add a similar check to the getMultipleNamespaces() validating that the returned IDs stay the same?

Comment (dimas-b) [reply servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/tests/AbstractRestNamespace.java:111]: `containsExactlyInAnyOrder` on line 116 uses `Namespace.equals()`, I assume, which takes ID into account.

Comment (snazy) [reply servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/tests/AbstractRestNamespace.java:111]: hm - yea - you're probably right ;)

Comment (snazy) [reply servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/tests/AbstractRestNamespace.java:111]: hm - yea - you're probably right ;)

Comment (dimas-b) [reply servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/tests/AbstractRestNamespace.java:111]: However, I added that check to the "update" test case and rebased.

**Review by snazy — APPROVED** [review]

**Pull request merged**